releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,11 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.1.0-preview - October 24th, 2024
+### Enhancements
+* Adds additional telemetry metadata field to feature flags
```

Comment:
Added support for injecting additional telemetry metadata field to feature flags if telemetry is enabled